cmd/earthly/build_cmd.go

```diff
@@ -553,16 +553,16 @@ func (app *earthlyApp) actionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs
 		select {
 		case <-cliCtx.Context.Done():
 			now := time.Now()
-			app.console.VerbosePrintf(
-				"========== CONTEXT DONE BEFORE LOGSTREAMER STARTED AT %s (%s later) ==========",
-				now.Format(time.RFC3339Nano),
-				now.Sub(beforeSelect),
-			)
+			app.console.VerbosePrintf("Logbus: hit context deadline at %s (%s later)", now.Format(time.RFC3339Nano), now.Sub(beforeSelect))
 			return
 		case details := <-buildOpts.MainTargetDetailsFuture:
+			if app.logbusSetup.OrgAndProjectSet() {
+				app.console.VerbosePrintf("Organization and project already set via environmental")
```

Comment:
In this case, do we perhaps want to still verify that the org and project match?

Reply:
I wasn't really sure of what the behavior ought to be. Should we fail the build or just print a message?

Reply:
I would fail the build in this case - I think it's better to fail early, rather than let the user get confused when the secrets don't work.